entrypoint.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eo
 
+echo "Cloning actions-collection@v1 repository..."
+git clone -b v1 https://github.com/variant-inc/actions-collection.git
+
```

Comment:
Do `source collection.sh` here run `docker_publish` function in [line 31](https://github.com/variant-inc/actions-dotnet/pull/16/files#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R31)

Reply:
Done.